ani-cli.1

```diff
@@ -0,0 +1,101 @@
+.\" Automatically generated by Pandoc 2.9.2.1
+.\"
+.TH "ANI-CLI (1) ani-cli" "" "\[lq]June 19 2022\[rq]" "" ""
+.hy
+.SH NAME
+.PP
+ani-cli - watch anime from the commandline
+.SH SYNOPSIS
+.PP
+\f[B]ani-cli\f[R] [-s] [-v] [-q ] [-a ] [-d | -p ] []
+.PD 0
+.P
+.PD
+\f[B]ani-cli\f[R] [-s] [-v] [-q ] -c
+.PD 0
+.P
+.PD
+\f[B]ani-cli\f[R] -h | -D | -U | -V
+.SH DESCRIPTION
+.PP
+\f[B]ani-cli\f[R] is a shell script to browse and search anime from the
+command-line.
+This tool scrapes the site animixplay.
+\f[B]ani-cli\f[R] without options defaults to the mpv media player.
+.SH OPTIONS
+.TP
+\f[B]-c\f[R]
+continue watching anime from history.
+.TP
+\f[B]-a\f[R] \[dq] episode\[dq]
+specify episode to watch.
+.TP
+\f[B]-h\f[R]
+Show summary of options.
+.TP
+\f[B]-d\f[R]
+download episode.
+.TP
+\f[B]-p\f[R] \[lq]download_dir\[rq]
+download episode to specified directory.
+.TP
+\f[B]-q\f[R] \[lq]quality\[rq]
+set video quality (best|worst|360|480|720|1080).
+.TP
+\f[B]-v\f[R]
+use VLC as the media player
+.TP
+\f[B]-s\f[R]
+watch anime together with friends, using Syncplay (works with mpv only)
+.PP
+\f[B]-d\f[R] download episode
```

Comment:
this needs a line break to be indented as the others:
![image](https://user-images.githubusercontent.com/66648475/208687421-ac2d8f7d-61c5-4a0e-a73b-59de0e167cd2.png)